### ISSUE TITLE
32bit version for new builds .. please senpai ?

### ISSUE THREAD

**Issue by Shinox** [Tinob/Ishiiruka#5]
Since the developers of dolphin stopped making 32bit builds , could you please without further ado make a at least 1 32bit build for the latest build from dolphin , that would mean a lot to me <3

**Comment by Zetchzie** [comment]
Yes plz do this. bring back the good dolphin before the devs ditched all us low spec users.

**Comment by phly95** [comment]
Can't you just download a 64 bit windows and install it on your current machine? 32 bit licenses work for 64 bit versions as well as long as you have Windows 7+

**Comment by lvivtotoro** [comment]
+1. 32-bits is enough for everybody.
